LoggingKata.Test/TacoParserTests.cs

```diff
@@ -1,29 +1,49 @@
 using System;
 using Xunit;
+using System.Collections.Generic;
 
 namespace LoggingKata.Test
 {
     public class TacoParserTests
     {
         [Fact]
         public void ShouldDoSomething()
-        {
-            // TODO: Complete Something, if anything
+        { 
+            Console.WriteLine("Hello Chip!");
         }
 
         [Theory]
-        [InlineData("Example")]
+        [InlineData("34.8831, -84.293899, Taco Bell Blue Ridg")]
+        [InlineData("30.39371, -87.68332, Taco Bell Fole")]

```

Comment:
Lol yeah, I was sitting next to Kris Dixon while I was working and since he finished he told me those were the final answers because I was lost. Still, IIRC I just had the locations commented off to the side so I could make sure the parser worked.

Reply:
That's fine- just checking